library/src/hiptensor.cpp

```diff
@@ -151,18 +151,17 @@ hiptensorStatus_t hiptensorInitTensorDescriptor(const hiptensorHandle_t*     han
         return HIPTENSOR_STATUS_NOT_INITIALIZED;
     }
 
-    if((lens == nullptr)
+    if((lens == nullptr && strides != nullptr)
```

Comment:
Need to allow lens == nullptr in TensorDescriptor. 

It means reducing a tensor to a scalar. Like [[1,2,3], [1,2,3]] ---sum reduce--> 12.

Reply:
Isn't a scalar lengths = [1] ?

Reply:
In CUDA, the lengths parameter is ignored when creating a tensor descriptor if the rank is 0. 
<img width="362" alt="image" src="https://github.com/user-attachments/assets/a05b1dd3-9860-4750-ae08-2c31353fe6cb">

When I tested, set `nmodeC == 0` and `extentC={10}` or `extentC={3,2}`, the output is always correct. So extentC is ignored. 

We do the same thing. 

When `nmodeC ==0`, `lenghts` is empty in a tensor descriptor.